### DISCUSSION TITLE
Python Version

### DISCUSSION THREAD

**Discussion by michahn01** [The-OpenROAD-Project/OpenROAD#3103] [General]
(I apologize for my ignorance in advance -- if an answer can be found in the documentation or elsewhere, please let me know, and I'll take this question down.)

What versions of Python is OpenROAD compatible with? Is any version of Python3 supported?

**Comment by maliberty** [comment]
"openroad -python" gives you access to OR with a python prompt.  Note that this is still in development and subject to a change and missing features.  See https://github.com/The-OpenROAD-Project/OpenROAD/issues/1812

openroad as a pip module has been discussed but is not currently available (see https://github.com/The-OpenROAD-Project/OpenROAD/pull/2311)
